cloud/linode/loadbalancers_test.go

```diff
@@ -2,6 +2,7 @@ package linode
 
 import (
 	"context"
+	stderrors "errors"
```

Comment:
nit: rename the k8s package and let the stdlib errors use the name

Reply:
I knew you'd come here and suggest this 😁 I chose this as I didn't want to touch unrelated code in this PR, but I guess it does make the end result cleaner.

Reply:
when you know a person :D it's only one call! feel free to ignore the nit tho i dont mind